content/en/blog/_posts/2024-12-11-cpumanager-strict-cpu-reservation.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: blog
+title: 'Kubernetes v1.32: New CPU Manager Static Policy Option: Strict CPU Reservation'
+draft: true
+date: 2024-12-11
+slug: cpumanager-strict-cpu-reservation
+author: >
+  [Jing Zhang](https://github.com/jingczhang) (Nokia)
+---
+
+In Kubernetes v1.32, after years of community discussion, we are excited to introduce `strict-cpu-reservation` option for the [CPU Manager static policy](/docs/tasks/administer-cluster/cpu-management-policies/#static-policy-options). This feature is currently in alpha and hidden by default hence needs be enabled explicitly.
+
+
+## Understanding the feature
+
+The CPU Manager static policy is used to reduce latency or improve performance. The `reservedSystemCPUs` defines an explicit CPU set for OS system daemons and kubernetes system daemons. This option is specifically designed for Telco/NFV use cases where uncontrolled interrupts/timers may impact the workload performance. you can use this option to define the explicit cpuset for the system/kubernetes daemons as well as the interrupts/timers, so the rest CPUs on the system can be used exclusively for workloads, with less impact from uncontrolled interrupts/timers. More details of this parameter can be found on the [Explicitly Reserved CPU List](/docs/tasks/administer-cluster/reserve-compute-resources/#explicitly-reserved-cpu-list) page.
```

Comment:
`reservedSystemCPUs` isn't specifically designed for telco use cases. It's also used in other contexts that want similar outcomes.

For example, you could use this in an air defense system or a cluster helping with civil radar and air traffic control. I have no examples to share but you sure could.

Reply:
Actually, the above writing is a direct copy and paste from the existing writing:
https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#explicitly-reserved-cpu-list

Agreed with your comment, I will re-phrase.